resources.yaml

```diff
@@ -226,8 +215,8 @@ deployment:
     count: 4 #4
     flavor: g1.c14m40g1d50
     group: compute_gpu
-    gpu_ready: true
-    docker_ready: true
+    image: gpu
+    docker: true
```

Comment:
Those keys refer to keys above in the config file, while things like "volume" are different correct? It would be nice to see that maybe from the bar name.

Reply:
Only "image" refers to the subkeys of "images". All others (including `docker`) do not.